wiki/Struct.md

```diff
@@ -32,7 +32,7 @@ tasks.
 #### read()
```

Comment:
I think we should actually remove these from the wiki. These are 10 years old now and the codebase diverged a bit - not even sure they would run anymore.

Reply:
OK, or some sort of stub. Let's merge this and then do that?

Reply:
Yup!